.github/workflows/build.yaml

```diff
@@ -26,7 +29,9 @@ jobs:
             override: true
 
       - name: Debug Build (default features)
-        run: cargo build --all --target=${{ matrix.target }}
+        run: |
+          git rev-list origin/master..$GITHUB_SHA | xargs -t -I % sh -c 'git checkout %; cargo build --all --target=${{ matrix.target }}'
```

Comment:
How does this exit if a commit is bad? i.e. did you test with a bad commit as well as all working?

Reply:
Good point, let me try with a bad commit and I'll keep you posted.

Reply:
It behaves as expected. Please have a look at https://github.com/sboeuf/cloud-hypervisor-1/runs/1214411101?check_suite_focus=true